### PR TITLE
feat(sdk): add Bedrock support to SDK

### PR DESCRIPTION
**Pull request by smartcontracts**
**Description**
Updates the SDK to add support for Bedrock. Includes a minor fix to the
build pipeline for the contracts-bedrock package. Removes some utilities
that are now exported as part of core-utils.

### CODE REVIEW

File: packages/sdk/src/adapters/eth-bridge.ts [hunk 1]
```diff
@@ -178,7 +178,10 @@ export class ETHBridgeAdapter extends StandardBridgeAdapter {
           amount,
           0, // L1 gas not required.
           '0x', // No data.
-          opts?.overrides || {}
+          {
+            ...omit(opts?.overrides || {}, 'value'),
```
Comment (roninjin10) [inline packages/sdk/src/adapters/eth-bridge.ts:182]: nit: the omit here is unnecessary it's being overwritten on line 183

File: packages/sdk/src/adapters/eth-bridge.ts [hunk 2]
```diff
@@ -187,7 +190,10 @@ export class ETHBridgeAdapter extends StandardBridgeAdapter {
...
```
Comment (roninjin10) [inline packages/sdk/src/adapters/eth-bridge.ts:194]: nit: omit unnecessary

**Review by roninjin10 — APPROVED** [review]
lgtm couple of nits but non blocking

**Review by mslipper — APPROVED** [review]

**Comment by mslipper** [comment]
This looks good to me. @tynes if you want another set of eyes please have them review, then remove `mergify-ignore`.

File: packages/sdk/src/adapters/eth-bridge.ts (hunk 1 above)
Comment (tynes) [reply packages/sdk/src/adapters/eth-bridge.ts:182]: Going to not change this as to be extra sure no behavior is changed in the old sdk version

Comment (tynes) [reply packages/sdk/src/adapters/eth-bridge.ts:182]: This will need to be refactored once its only bedrock support anyways

**Pull request merged**